autoload/asyncomplete/sources/flow.vim

```diff
@@ -9,9 +10,13 @@ function! asyncomplete#sources#flow#completor(opt, ctx) abort
     let l:tempfile = s:write_buffer_to_tempfile(a:ctx)
 
     let l:config = get(a:opt, 'config', {})
-    let l:flowbin_path = get(l:config, 'flowbin_path', 'flow')
+    if get(l:config, 'prefer_local', 0)
+        let l:flowbin_path = s:resolve_flowbin_path(l:file, get(l:config, 'flowbin_path', 'flow'))
```

Comment:
if `pefer_local` is one, I wouldn't use `config.flowbin_path` since it is most likely always going to be absolute path.